salt/cloud/clouds/tencentcloud.py

```diff
@@ -0,0 +1,1029 @@
+# -*- coding: utf-8 -*-
+
+'''
+Tencent Cloud Cloud Module
+=============================
+
+.. versionadded:: 2019.09.01
```

Comment:
3000?

Reply:
:+1:  It looks like the linter might have an issue with 3000 as a version